src/renderer/components/analytics/components/ui/Tabs.tsx

```
@@ -0,0 +1,55 @@
+
+  return (
+    <Box sx={{ mb: 2 }}>
+      <MuiTabs
+        value={Math.min(activeTab, tabs.length - 1)}
+        onChange={(_, v) => setActiveTab(v)}
+        variant="scrollable"
+        scrollButtons="auto"
+      >
+        {tabs.map((tab, i) => (
+          <MuiTab
+            key={i}
```

Biome (2.5.0)

[error] Avoid using the index of an array as key property in an element.

(lint/suspicious/noArrayIndexKey)